railties/lib/rails/generators/rails/app/templates/config/databases/trilogy.yml.tt

```diff
@@ -51,6 +51,12 @@ test:
 #   production:
 #     url: <%%= ENV["MY_APP_DATABASE_URL"] %>
 #
+# The connection URL for non-primary databases can also be configured using a
+# similar environment variable prefixed with the configuration key in uppercase.
+# For example:
+#
+#   CACHE_DATABASE_URL="trilogy://cacheuser:cachepass@localhost/cachedatabase"
+#
```

Comment:
I think this is generally a good change, but it seems unnecessary with `--skip-solid`. What do you think about making it conditional on the solid databases being used?

Reply:
Makes sense to me. I'm AFK
at the moment, but I'll update the branch as soon as I have time - probably tomorrow morning.

Reply:
Addressed in https://github.com/rails/rails/pull/54765/commits/8a11d7c55d17e584140951ed9e2af53f0e6d72d6. I've also updated the PR description & the commit note. Marking as resolved.